### PR TITLE
feat(EMS-3159): No PDF - Export contract - Check/change answers - Agent charges 

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:
This PR adds the ability for a user to change their answers for the "agent charges" questions in the "Export contract" section of a no PDF application.

## Resolution :heavy_check_mark:
- Add E2E tests for:
  - When the charge `METHOD` is `FIXED_SUM`:
    - Changing the `FIXED_SUM_AMOUNT`
    - Changing the `METHOD`  to `PERCENTAGE`.
  - When the charge `METHOD` is `PERCENTAGE`:
    - Changing the `PERCENTAGE_CHARGE`
    - Changing the `METHOD`  to `FIXED_SUM`.
  - Changing the `PAYABLE_COUNTRY_CODE`.
- Update `test.yml`.
- Update `mapSubmittedData` to map and nullify fields depending on the `METHOD`.

### CODE REVIEW

File: e2e-tests/insurance/cypress/e2e/journeys/export-contract/change-your-answers/agent-charges/change-your-answers-agent-charges-fixed-sum-amount.spec.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+const {
+  AGENT_CHARGES: { FIXED_SUM_AMOUNT },
+} = FIELD_IDS;
+
+const fieldId = FIXED_SUM_AMOUNT;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Export contract - Change your answers - Agent charges - As an Exporter, I want to be able to review my input regarding the amount an agent is charging for helping me win my export contract, So that I can be assured I am providing UKEF with the right information', () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/export-contract/change-your-answers/agent-charges/change-your-answers-agent-charges-fixed-sum-amount.spec.js:22]: could we add what is being changed in the context? like `Agent charges - Fixed sum amount - `

Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/export-contract/change-your-answers/agent-charges/change-your-answers-agent-charges-fixed-sum-amount.spec.js]: Great shout, have updated all related tests

File: e2e-tests/insurance/cypress/e2e/journeys/export-contract/change-your-answers/agent-charges/change-your-answers-agent-charges-fixed-sum-amount.spec.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+const {
+  AGENT_CHARGES: { FIXED_SUM_AMOUNT },
+} = FIELD_IDS;
+
+const fieldId = FIXED_SUM_AMOUNT;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context(`Insurance - Export contract - Change your answers - Agent charges - ${FIXED_SUM_AMOUNT} As an Exporter, I want to be able to review my input regarding the amount an agent is charging for helping me win my export contract, So that I can be assured I am providing UKEF with the right information`, () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/export-contract/change-your-answers/agent-charges/change-your-answers-agent-charges-fixed-sum-amount.spec.js:22]: not a blocker, missing - after FIXED_SUM_AMOUNT

**Review by Zainzzkk — APPROVED** [review]

File: e2e-tests/insurance/cypress/e2e/journeys/export-contract/change-your-answers/agent-charges/change-your-answers-agent-charges-fixed-sum-amount.spec.js (hunk 2 above)
Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/export-contract/change-your-answers/agent-charges/change-your-answers-agent-charges-fixed-sum-amount.spec.js:22]: Good spot, will fix in the next PR

**Pull request merged**